### ISSUE TITLE
Panic if you try to run a query and have a comment on the previous line

### ISSUE THREAD

**Issue by kyruzic** [achristmascarl/rainfrog#32]
Running this

```sql
-- hi

SELECT * FROM "Table";
```

will always produce this panic

```
Well, this is embarrassing.

rainfrog had a problem and crashed. To help us diagnose the problem you can send us a crash report.

We have generated a report file at "/tmp/report-c8a60be1-994f-4dde-a56f-2ff2f33c6a69.toml". Submit an issue or email with the subject of "rainfrog Crash Report" and include the report as an attachment.

- Authors: achristmascarl <carl@rainfrog.dev>

We take privacy seriously, and do not perform any automated error collection. In order to improve the software, we rely on people to submit reports.

Thank you kindly!
The application panicked (crashed).
Message:  index out of bounds: the len is 0 but the index is 0
Location: src/database.rs:119

This is a bug. Consider reporting it at https://github.com/achristmascarl/rainfrog
```

**Comment by achristmascarl** [comment]
this is fixed in [v0.1.10](https://github.com/achristmascarl/rainfrog/releases/tag/v0.1.10), thanks for the report!

**Comment by achristmascarl** [comment]
did not fix it for when there was a semi-colon in the comment; will change query parsing to be more robust
